### DISCUSSION TITLE
Can VLang be used in a production environment?

### DISCUSSION THREAD

**Discussion by male110** [vlang/v#19286] [Questions and Answers]
Can VLang be used in a production environment? Are there any real-world examples of it being used in production?

**Comment by larpon** [comment]
Yes, I use this daily without problems:
https://github.com/Larpon/tipsy

And have released a game written in V that is stable and playable:

https://blackgrain.itch.io/puzzle-vibes

Since V has not reached version 1.0 there might be bugs here and there but they tend to get fixed very quickly though. I personally find V very stable for daily use
